### ISSUE TITLE
Add new utils.js functions for converting temperature units from C to F and vice versa

### ISSUE THREAD

**Issue by judemorrissey** [judemorrissey/rn-fun#38]
@semamama assigning you this issue, please checkout the remote branch named `add-celsius-fahrenheit-conversion-units` [(see the branch on github here)](https://github.com/judemorrissey/rn-fun/tree/add-celsius-fahrenheit-conversion-utils) and fill in the functions i stubbed in the `utils.ts` file.

As a reminder, you can check out the remote branch locally by using the following commands:
- `git fetch -p` updates known remote branches and commits, and the `-p` option prunes any remote branches that no longer exist from your local
- `git checkout add-celsius-fahrenheit-conversion-utils` makes a new local branch named the same as the remote, and switches to it

Once you are in the new branch, I stubbed out two functions named `convertCelsiusToFahrenheit` and `convertFahrenheitToCelsius` in `src/lib/utils/utils.ts`. Fill in those two functions, then you can run the new tests I wrote in `src/lib/utils/test.ts` to check your work.
- `yarn run utils` will run tests on matching files that contain `*utils*` in its path

After writing the two function bodies and verifying tests work, push your changes to the remote branch
- remember, `git add .` and `git commit -m 'useful message here'` as easy ways to make a new commit with your changes
- `git push` will push your local changes to the remote branch, note that this is the shorthand because by checking out a remote branch locally, git automatically sets up remote tracking for the local branch

Once that is done, go to the github web UI and make a new pull request for the branch, and assign me as code reviewer. You can refer to the [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) if you need.
